### PR TITLE
Binalyze air

### PR DESCRIPTION
**Pull request by binalyze-murat**
## Status
- [] In Progress
- [x ] Ready
- [] In Hold - (Reason for hold)

## Related Issues
No fix is available at this time. 

## Description
This is the first commit of Binalyze AIR integration.
## Minimum version of Cortex XSOAR
- [x ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x ] No

## Must have
- [x ] Tests
- [x ] Documentation

### CODE REVIEW

**Review by AradCarmi — CHANGES_REQUESTED** [review]
Hey Murat,
You did a great job!
I didn't finish reviewing all the files, only the py file, I will let you view my notes and suggestions, and accordingly, you can change the other files. Let me know when you are finished and we keep moving forward in order to merge the PR.
Again, thanks for the contribution!

File: Packs/Binalyze/Integrations/BinalyzeAIR/BinalyzeAIR.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+def test_connection(base_url) -> str:
+    '''Command for test-connection'''
+
+    response = requests.get(f'{base_url}/api/app/info')
+    if response.status_code == 200:
+        return demisto.results('ok')
+    else:
+        return demisto.results('test connection failed')
+
```
[the remark this answers is not in the document]
Comment (binalyze-murat) [reply Packs/Binalyze/Integrations/BinalyzeAIR/BinalyzeAIR.py]: this is done as well

File: Packs/Binalyze/Integrations/BinalyzeAIR/test_data/acquire.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (AradCarmi) [inline Packs/Binalyze/Integrations/BinalyzeAIR/test_data/acquire.json:1]: Seems it has never been used.

File: Packs/Binalyze/Integrations/BinalyzeAIR/test_data/isolate.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (AradCarmi) [inline Packs/Binalyze/Integrations/BinalyzeAIR/test_data/isolate.json:1]: Seems it has never been used.

File: Packs/Binalyze/Integrations/BinalyzeAIR/BinalyzeAIR_test.py [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+
+MOCK_URL = 'https://nonexistent-domain.com/'
+
+
+def test_connection_success(requests_mock: Any) -> None:
+    '''Command for test-connection'''
+
+    expected_mocked_command_result: str = 'ok'
+    requests_mock.get(f'{MOCK_URL}/api/app/info')
+    assert expected_mocked_command_result
```
Comment (AradCarmi) [inline Packs/Binalyze/Integrations/BinalyzeAIR/BinalyzeAIR_test.py:12]: The tests overall seem good, my only suggestion is to use the `client.test_conectivity` in the test_connection_* tests. 
(You just asserted that the expected_mocked_command_result is not None which is always true)

**Review by AradCarmi — CHANGES_REQUESTED** [review]
Hey Murat, Nicely done! 
These are the last changes before we merge the PR. Please view my notes and resolve each one, if you are not sure just add a comment with the question or send a message via Slack. The Unit Test looks great (see my note there) if the coverage will not raise we will ignore this validation, so don't worry 🙃

**Comment by AradCarmi** [comment]
There are still some suggestions that you didn't resolve.

File: Packs/Binalyze/Integrations/BinalyzeAIR/BinalyzeAIR.py [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+        demisto.debug(f'Command being called is {demisto.command()}')
+        client: Client = Client(
+            base_url=base_url,
+            verify=verify_certificate,
+            headers=headers,
+            proxy=proxy,
+            ok_codes=(404, 200)
+        )
+        if command == 'test-module':
+            return_results(test_connection(client))
```
[the remark this answers is not in the document]
Comment (binalyze-murat) [reply Packs/Binalyze/Integrations/BinalyzeAIR/BinalyzeAIR.py:152]: I changed the code as base_url will be called from Client class. I think this fix is not needed anymore.

File: Packs/Binalyze/Integrations/BinalyzeAIR/BinalyzeAIR.yml [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+    - contextPath: BinalyzeAIRIsolate.result.organizationId
+      description: Organization Id of endpoint
+      type: number
+    - contextPath: BinalyzeAIRIsolate.statusCode
+      description: HTTP Status Code of response
+      type: number
+    - contextPath: BinalyzeAIRIsolate.errors
+      description: Error message
+      type: string
+    description: Isolate an endpoint
```
[the remark this answers is not in the document]
Comment (binalyze-murat) [reply Packs/Binalyze/Integrations/BinalyzeAIR/BinalyzeAIR.yml]: > In general, I recommend removing the success, statusCode, and errors.

done

[the remark this answers is not in the document]
Comment (binalyze-murat) [reply Packs/Binalyze/Integrations/BinalyzeAIR/BinalyzeAIR.yml]: > Also please add a dot as suggested to all of the outputs.
done

File: Packs/Binalyze/Integrations/BinalyzeAIR/README.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
+## Binalyze AIR Integration
```
[the remark this answers is not in the document]
Comment (binalyze-murat) [reply Packs/Binalyze/Integrations/BinalyzeAIR/README.md:1]: done

File: Packs/Binalyze/Integrations/BinalyzeAIR/test_data/acquire.json (hunk 2 above)
Comment (binalyze-murat) [reply Packs/Binalyze/Integrations/BinalyzeAIR/test_data/acquire.json]: done, deleted.

**Review by AradCarmi — APPROVED** [review]

**Pull request merged**